tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -63,12 +63,12 @@
 
     <!-- Integration: Google.Protobuf -->
     <!--    Assembly: Google.Protobuf -->
-    <!-- Latest package https://www.nuget.org/packages/Google.Protobuf/3.28.0 -->
-    <PackageReference Include="Google.Protobuf" Version="3.28.0" />
+    <!-- Latest package https://www.nuget.org/packages/Google.Protobuf/3.28.2 -->
+    <PackageReference Include="Google.Protobuf" Version="3.28.2" />
 
     <!-- Integration: GraphQL -->
     <!--    Assembly: GraphQL -->
-    <!-- Latest package https://www.nuget.org/packages/GraphQL/8.0.2 -->
+    <!-- Latest package https://www.nuget.org/packages/GraphQL/8.1.0 -->
     <PackageReference Include="GraphQL" Version="7.9.0" />
```

Comment:
We need to add support here for a new major version: @bouwkast @lucaspimentel

Reply:
Noted, I'll get a PR going for it